NuGet.config

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <clear />
```

Comment:
We shouldn't do this. We want to make sure that only internal feeds are used for package restore. Instead, we should push the UnitesNet package into our internal feeds.

Reply:
The new build servers don't complain, so....

I can't upload there, so I need to do this to get the build to run. But I agree, it's not really nice.